test/realm_test.dart

```diff
@@ -53,7 +53,7 @@ class _Dog {
 @RealmModel()
 class _Team {
   late String name;
-  late List<_Person> players;
+  final players = <_Person>[];
```

Comment:
as I commented above I am not sure we need this when we generate a getter for lists

Reply:
the alternative is `late final List<_Person> player;` which is actually longer. The generator supports both.

Reply:
Changed